longevity_oos_test.py

```
@@ -79,6 +79,20 @@ def ignore_repair_errors():
         yield
 
 
+@contextmanager
+def ignore_runtime_errors():
+    with ExitStack() as stack:
+        stack.enter_context(
+            EventsSeverityChangerFilter(
+                new_severity=Severity.NORMAL,
+                event_class=DatabaseLogEvent,
+                regex=".*Failed to load SSTable.*due to std::runtime_error \(critical disk utilization\).*",
```

Ruff (0.16.0)

[warning] Invalid escape sequence: `\(`

Use a raw string literal

(W605)